deploy-preflight.yml clones private protocols repo unauthenticated → exit 128 (blocks all wired deploys)

## Severity: blocks deploys
First real execution of `deploy-preflight.yml` (wired into qwickwatch deploy.yml via qwickwatch#240) fails. It blocks the deploy path for any repo that wires the gate.

## Root cause
The reusable workflow step:
```
- name: Checkout protocols scripts
  run: git clone --depth 1 https://github.com/qwickapps/protocols .protocols
```
clones `qwickapps/protocols`, which is **PRIVATE**, with **no credentials**. On the self-hosted macmini runner the Actions job env has no git credential helper, so:
```
Cloning into '.protocols'...
fatal: could not read Username for 'https://github.com': Device not configured
##[error]Process completed with exit code 128.
```
(Run: qwickapps/qwickwatch run 26518085310, preflight job.)

## Fix (pick one)
- Use `actions/checkout@v4` with `repository: qwickapps/protocols`, `path: .protocols`, and a `token:` that can read protocols (a PAT or GitHub App token stored as an org secret — NOT the caller's `GITHUB_TOKEN`, which is scoped to the caller repo only).
- Or clone with an embedded token: `git clone https://x-access-token:${{ secrets.PROTOCOLS_RO_TOKEN }}@github.com/qwickapps/protocols .protocols`.

Requires provisioning a read-only token/secret with access to protocols — a fleet/secrets decision.

## Verification
After the fix, re-run qwickwatch deploy (or any wired repo) and confirm the preflight job reads the contract + passes, then build/deploy proceed.

--- a/.github/workflows/deploy-preflight.yml
+++ b/.github/workflows/deploy-preflight.yml
@@ -18,13 +18,16 @@ jobs:
       - name: Checkout caller repo
         uses: actions/checkout@v4
 
-      - name: Checkout protocols scripts
-        run: git clone --depth 1 https://github.com/qwickapps/protocols .protocols
+      - name: Verify protocols runtime available
+        run: |
+          if [[ ! -f ~/.qwickapps/protocols/scripts/preflight-deploy-contract.sh ]]; then
+            echo "::error::~/.qwickapps/protocols/scripts/preflight-deploy-contract.sh not found. Ensure install-fleet CI ran successfully."
+            exit 1
+          fi
 
       - name: Install PyYAML
         run: python3 -m pip install --user PyYAML --quiet 2>/dev/null || true
 
       - name: Run deploy contract preflight
         run: |
-          chmod +x .protocols/scripts/preflight-deploy-contract.sh
-          .protocols/scripts/preflight-deploy-contract.sh
+          ~/.qwickapps/protocols/scripts/preflight-deploy-contract.sh

